service_info/static/js/src/index.js

```diff
@@ -22,4 +42,11 @@ jQuery(function ($) {
   if (!$('.cms-toolbar-expanded').length) {
     $('.parallax').parallax();
   }
+
+  /*
+    Adjust for IE 11.
+  */
+  if (getInternetExplorerVersion() !== -1) {
```

Comment:
seems like rv should be initialized to NaN and this should compare against NaN, since parseFloat() returns NaN on error; then, something that unexpectedly passes through the regex won't be treated like InternetExplorer